controllers/util.go

```diff
@@ -180,10 +174,10 @@ func ensureDeletePipelineIsReconciled(jobOpts jobOpts) (ctrl.Result, error) {
 		if err := jobOpts.client.Update(jobOpts.ctx, jobOpts.obj); err != nil {
 			return ctrl.Result{}, err
 		}
+	} else {
+		jobOpts.logger.Info("Delete Pipeline not finished", "status", existingDeletePipeline.Status)
```

Comment:
💅 This is likely polish, but would it be helpful to also log an info on success? I ask because if delete succeeds the k8s resources disappear. It may be nice to have seen it in the logs 💭

Reply:
Do you mean we should log if the delete pipeline completed successfully? We had this already (a few lines above):

`jobOpts.logger.Info("Delete Pipeline Completed")`